packages/inter-protocol/test/auction/README-bidding-cli.md

```diff
@@ -0,0 +1,278 @@
+bid is broadcast:
+{
+  "timestamp": "2023-03-31T05:46:27Z",
+  "height": "54871",
+  "offerId": "bid-1680241587424",
+  "txhash": "CC8D778F51867B36CB91D0..."
+}
+{
+  "id": "bid-1680241587424",
+  "price": "8.50 IST/IbcATOM",
+  "give": { "Currency": "85IST" },
+  "result": "Your bid has been accepted"
+}
+```
+
+Then perhaps yours is the winning bid:
+
+```
+$ inter bid list --from test-acct --all | jq
+{
+  "id": "bid-1680241587424",
+  "payouts": {"Collateral":"3.105 ATOM"}
+}
+```
+
+See also [test-inter-cli.js.md](https://github.com/Agoric/agoric-sdk/blob/master/packages/agoric-cli/test/snapshots/test-inter-cli.js.md) for exhaustive command options.
+
+## Installation
+
+You will need to install several components before you’ll be in a position to participate in liquidations bidding. These include:
+-The AgoricSDK and prerequisite dev tools 'node', 'go', 'yarn', 'agd' (CLI tool) and 'agops'.
+
+Conveniently you can install all these using:
+
+```sh
+git clone https://github.com/Agoric/agoric-sdk
+cd agoric-sdk
+SKIP_DOWNLOAD=false ./bin/agd build
+```
+
+
+### Locating the `agd` command
+
+`agd` follows `go` conventions, so be sure `~/go/bin` is in your `$PATH`:
+
+```sh
+export PATH=~/go/bin:$PATH
+```
+
+Then try `agd version` to confirm.
+
+### Locating the `agops inter` command
+
+`agops` follows `npm`/`yarn` conventions, so you can `yarn run agops inter` as long as your current directory is under `agoric-sdk`.
+
+#### Add an alias for `agops inter`
+
+```sh
+alias inter="yarn run --silent agops inter"
+```
+
+Then you can just run...
+
+```
+$ inter
+Usage: agops inter [options] [command]
+
+Inter Protocol commands for liquidation bidding etc.
+...
+```
+
+
+## Key management with `agd`
+
+If you don't already have a testing address and corresponding private key:
+
+```sh
+agd keys add test-acct --keyring-backend=test
+```
+
+For example:
+
+```
+$ agd keys add test-acct --keyring-backend=test
+
+- name: test-acct
+  type: local
+  address: agoric18jr9nlvp300feu726y3v4n07ykfjwup3twnlyn
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"ApPNaneZjAWy2RarLaIw5O3vkBD6R3uZ9F2eRogei2BO"}'
+  mnemonic: ""
+
+
+**Important** write this mnemonic phrase in a safe place.
+It is the only way to recover your account if you ever forget your password.
+
+abandon plate bind ...
+```
+
+**CAUTION**: `agd keys add` displays the private mnemonic.
+
+
+Using the `test` keyring is lower friction, but feel free to use the os keyring:
+
+```sh
+agd keys add test-acct
+```
+
+## Provisioning your account for use with Zoe smart contracts
+
+Mainnet
+
+You’ll need to provision an Agoric Smart Wallet account (offer handler) in order to interact with Zoe contracts. You’ll need acquire 10BLD in order to do this. See[ ‘How to Provisions a Smart Wallet instructions’](https://docs.inter.trade/user-how-to/wallet-usage/create-your-agoric-account-and-smart-wallet.). Once provisioned, you’ll need to hold an IST balance in order to be able to place a bid.
```

Comment:
The deleted period is not very visible in the diff, but it's the most semantically significant change below. The link won't work without the fix.

```suggestion
You’ll need to provision an Agoric Smart Wallet account (offer handler) in order to interact with Zoe contracts. You’ll need acquire 10BLD in order to do this. See [‘How to Provision a Smart Wallet instructions’](https://docs.inter.trade/user-how-to/wallet-usage/create-your-agoric-account-and-smart-wallet). Once provisioned, you’ll need to hold an IST balance in order to be able to place a bid.
```

Reply:
Note that I reverted the Mainnet stuff.